driver/glu_driver.hpp

```diff
@@ -0,0 +1,477 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#pragma once
+
+#include "InputFlags.hpp"
+#include "driver.hpp"
+#include "tensor_driver.hpp"
+#include "timer.hpp"
+#include "random.hpp"
+
+#include <cstdint>
+#include <cstdlib>
+#include <limits>
+#include <memory>
+#include <vector>
+
+#include <../test/verify.hpp>
+
+#include <miopen/errors.hpp>
+#include <miopen/miopen.h>
+
+template <typename T>
+T sigmoid(T x)
+{
+    return 1.0f / (1.0f + exp(-x));
+}
+
+template <typename Tgpu, typename Tcheck>
+int mloGLUForwardContiguousRunHost(const Tgpu* input,
+                                   miopenTensorDescriptor_t outputDesc,
+                                   Tcheck* outputHost)
```

Comment:
I've updated this function to work with specified dimension

Reply:
I apologize, perhaps we had a misunderstanding. I do not insist that the function be universal, it would be enough to rename it correctly. As far as I understand, this functionality is not required at the moment. If you do not have a proven solution, then we could leave everything as is. But if there is a solution, then I am not against implementing it.